### PR TITLE
Queries for residential heating technology chart

### PR DESCRIPTION
**Pull request by KoenvanB**
This PR adds queries for the residential heating technology chart. See [PR #4216](https://github.com/quintel/etmodel/pull/4216) in ETModel. The PR includes:

- queries to calculate the number of residences for each space heating technology, ordered by build year;
- queries to calculate the total number of residences for a build year class (currently unused)

**Discussion**

- Is the systematic naming of the queries in line with existing queries? E.g. is a distinction required between 'households' and 'residences'?
- Should the `number_of_residences_<build_period>` queries need to be based on the build year node groups?

### CODE REVIEW

File: gqueries/output_elements/output_series/vertical_stacked_bar_number_of_households_per_space_heating_technology/number_of_residences_per_build_year/number_of_residences_1945_1964.gql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- unit = #
+- query = 
+    SUM(
+        V(households_useful_demand_for_space_heating_apartments_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_detached_houses_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_semi_detached_houses_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_terraced_houses_1945_1964, number_of_units),
+    )
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/vertical_stacked_bar_number_of_households_per_space_heating_technology/number_of_residences_per_build_year/number_of_residences_1945_1964.gql:8]: Wouldn't using the node groups be easier? For example: `SUM(V(G(households_1945_1964),number_of_units))`

File: gqueries/output_elements/output_series/vertical_stacked_bar_number_of_households_per_space_heating_technology/space_heater_coal/households_1945_1964_with_space_heater_coal.gql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        V(households_useful_demand_for_space_heating_apartments_1945_1964,number_of_units)
+      )
+    )
+
+
+
+
+
+
+
```
Comment (mabijkerk) [inline gqueries/output_elements/output_series/vertical_stacked_bar_number_of_households_per_space_heating_technology/space_heater_coal/households_1945_1964_with_space_heater_coal.gql:27]: These additional returns seem unnecessary.

**Comment by mabijkerk** [comment]
I think it would be very useful to write a short mechanical turk spec for this. For example, one test to check if all the queries per technology sum to the number of units of that technology and one test to check if all the queries in the chart sum to the total number of residents.

**Comment by KoenvanB** [comment]
@mabijkerk I have 

- deleted redundant empty lines in all files;
- used the node groups to calculate the number of residences per build year;
- moved those queries to general;
- changed the names of all queries for the chart to refer to `number_of_houses` instead of `households`

That makes this PR ready to merge

**Pull request merged**